doc/api/buffer.md

```diff
@@ -403,7 +403,7 @@ The `size` must be less than or equal to the value of
 be created if a `size` less than or equal to 0 is specified.
 
 If `fill` is specified, the allocated `Buffer` will be initialized by calling
-`buf.fill(fill)`. See [`buf.fill()`][] for more information.
+`buf.fill(fill)`. See [`buf.fill(0)`][] for more information.
```

Comment:
Could you change only the href here, not the text itself? I think that makes more sense.